pgx-pg-sys/build.rs

```diff
@@ -317,33 +312,24 @@ fn impl_pg_node(items: &Vec<syn::Item>) -> eyre::Result<Vec<syn::Item>> {
         let struct_name = &node_struct.struct_.ident;
 
         // impl the PgNode trait for all nodes
-        pgnode_impls.push((
-            struct_name.to_string(),
-            syn::parse2(quote! {
-                impl pg_sys::PgNode for #struct_name {
-                    type NodeType = #struct_name;
-                }
-            })?,
-        ));
+        pgnode_impls.extend(quote! {
+            impl pg_sys::PgNode for #struct_name {
+                type NodeType = #struct_name;
+            }
+        });
 
         // impl Rust's Display trait for all nodes
-        pgnode_impls.push((
-            struct_name.to_string(),
-            syn::parse2(quote! {
+        pgnode_impls.extend(
+            quote! {
                 impl std::fmt::Display for #struct_name {
                     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
                         write!(f, "{}", crate::node_to_string_for_display(self.as_node_ptr() as *mut crate::Node))
                     }
                 }
-            })?,
-        ));
+            });
     }
 
-    // sort the pgnode impls by their struct name so that we have a consistent ordering in our bindings output
-    pgnode_impls.sort_by(|(a_name, _), (b_name, _)| a_name.cmp(b_name));
```

Comment:
Do we want to maintain this sorting behavior to help reduce diffs on binding generation?

Reply:
I thought about it, but decided it doesn't matter.  The diffs for the bindings files are typically large anyways when we do change them.  I was also trying to just eliminate some overhead during build.rs.

You can convince me otherwise

Reply:
I'd prefer to keep the generated diffs each release commit as small as we can, and if there are changes, I prefer them to be meaningful. This is useful for tracking possible regressions. 

... How about we only sort the bindings we write for docs.rs?

Reply:
> .. How about we only sort the bindings we write for docs.rs?

oh, great idea!

Reply:
Committed in https://github.com/tcdi/pgx/pull/526/commits/8021953305de4a86bc2a6af8e8f7a57b44c42e1b